pylint/checkers/classes/class_checker.py

```diff
@@ -1547,7 +1568,7 @@ def _check_invalid_class_object(self, node: nodes.AssignAttr) -> None:
             return
         self.add_message("invalid-class-object", node=node)
 
-    def _check_in_slots(self, node):
+    def _check_in_slots(self, node: nodes.AssignAttr) -> None:
```

Comment:
This seems to be `SuccessfulInferenceResult`?

Reply:
Hmm, I think the function never return anything ? I see only empty ``return`` or do you mean ``node`` ?